docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,241 @@
+
+For example, we configured a person detector as follows:
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+  {
+    "type": "mlmodel",
+    "model": "tflite_cpu",
+    "attributes": {
+      "model_path": "${packages.persondetect0tflite}/persondetect0.tflite",
+      "label_path": "${packages.persondetect0tflite}/persondetectlabels.txt",
+      "num_threads": 1
+    },
+    "name": "persondetect"
+  },
+  {
+    "name": "people-detect",
+    "type": "vision",
+    "model": "mlmodel",
+    "attributes": {
+      "mlmodel_name": "persondetect"
+    }
+  }
+]
+```
+
+This will serve as the first and second layers of your verification system.
+Now, move on to configuring the "finer" layer of the detectors-- the facial recognition detector.
+
+## Configure a facial detector
+
+Now, to disarm our alarm, we want our verification system to be able to give someone an "ok" based on their ID.
+To create a detector that can recognize individual faces, use Viam Lab's `facial-detector` module, available in the [registry](https://app.viam.com/module/viam-labs/facial-detector).
+This is a modular vision service that uses the DeepFace library to perform facial detections.
+
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/).
+Click on the **Services** subtab and click **Create service**.
+Select the `vision` type, then select the `detector:facial-detector` model. Enter a name for your vision service, `"face-detect"` and click **Create**.
+
+Edit the attributes as applicable according to the configuration information on [GitHub](https://github.com/viam-labs/facial-detection):
+
+- `"face_labels"`: Label a photo of the face of each person you want your security system to recognize with the name you want for the label paired with the image path.
+- `"recognition_model"`: The model to use for facial recognition. `"ArcFace"` is chosen as the default for a good balance of speed and accuracy. See [GitHub](https://github.com/viam-labs/facial-detection) for more options.
+- `"detection_framework"`: The detection framework to use for facial detection. `"ssd"` is chosen as the default for a good balance of speed and accuracy.
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "face_labels": {
+    "my_name": "/home/me/my-photo.jpg"
+  },
+  "recognition_model": "ArcFace",
+  "detection_framework": "ssd"
+}
+```
+
+Make sure to name this detector `"face-detect"`.
+
+## Configure a verification system
+
+Now that you've configured your camera, a rough `"people-detect"` detector and a more fine-tuned `"face-detect"` facial detector, you can set up your facial verification system module.
+
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Services** subtab and click **Create service**. Select the `vision` type, then select the `classifier:verification-system` model. Enter a name for your vision service and click **Create**.
```

Comment:
Please add more context for what adding this does (I go a bit into that in the above comments - it's adding the code that makes all of this work)

Reply:
Added above